programs/ssl/ssl_mail_client.c

```diff
@@ -493,25 +494,26 @@ int main( int argc, char *argv[] )
 
 #if defined(MBEDTLS_FS_IO)
     if( strlen( opt.ca_file ) )
-        ret = mbedtls_x509_crt_parse_file( &cacert, opt.ca_file );
+        retval = mbedtls_x509_crt_parse_file( &cacert, opt.ca_file );
     else
-#endif
+#endif /* MBEDTLS_FS_IO */
```

Comment:
Don't add this comment here, at least not in this commit. Our style guide only requires the `#endif` comment when the body of the `#if` is large or complicated.